lms/sql_tasks/tasks/report/create_from_scratch/04_activity_counts/03_organization_activity/01_create_view.sql

```diff
@@ -119,16 +141,23 @@ CREATE MATERIALIZED VIEW report.organization_activity AS (
             SELECT
                 period, timescale, organization_id, 'user'::report.roles AS role,
                 annotation_count, 0 AS active, 0 AS billable
-            FROM annotation_counts
+            FROM user_annotation_counts
 
-            UNION
+            UNION ALL
+
+            SELECT
+                period, timescale, organization_id, 'teacher'::report.roles AS role,
+                annotation_count, 0 AS active, 0 AS billable
+            FROM teacher_annotation_counts
+
+            UNION ALL
 
             SELECT
                 period, timescale, organization_id, role,
                 0, active, 0
             FROM active
 
-            UNION
+            UNION ALL
```

Comment:
We can use UNION ALL's here, which prevents Postgres from having to check each row for uniqueness. By the way we've constructed this here we know no rows can be duplicates, as each has something different in it.